### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by DavidParkin** [tshatrov/scriptfu#2]
Is there any?

**Comment by tshatrov** [comment]
Animstack documentation is here: http://tshatrov.github.io/animstack

Also the .htm files in this repository are html documentation from GIMP registry. They can be viewed either by being downloaded and opened in a browser, or like this:

https://htmlpreview.github.io/?https://github.com/tshatrov/scriptfu/blob/master/bgmask.htm

https://htmlpreview.github.io/?https://github.com/tshatrov/scriptfu/blob/master/export-layers-plus.htm

For layerscript I haven't bothered to write any documentation yet.

**Comment by DavidParkin** [comment]
Thanks, I found the github thingy but it was heavy going for a noob. I could have done with some examples. The videos were quite a jump from the manual beachball thingy to the artwork.

What I was aiming to do was move a shape about. I got this to work:

> shape[fg][+move:10:10]
Background [*24]

Worked backwards. Still trying to get it right. Any tips welcome.

GIMP Registry appears to be closed.

**Comment by tshatrov** [comment]
Yes, [fg] tag, unlike most other tags, works downwards the frame list, which means backwards in time. However there's a tilde modifier which makes the tag work in opposite direction. So `[~fg]` works upwards the frame list and forwards in time (`[copy:0]` is another way to do the same thing). However the fg layer would need to be positioned below the frames in this case, which might make it harder to work with (because background will be obscuring the foreground).

Generally I'm using transparent layers for frames and build up the result from there in natural order. For example:
```
Frame [*24]
shape[~fg][+move:10:10]
Background[bg]
```

**Comment by DavidParkin** [comment]
I got this to work.

> 
Background[*24]]
shape[copy:0][+move:10:10]

I did have problems with the background obscuring the shape. Will try your method.
